src/styles/dist/landing.css

```diff
@@ -1322,4 +1326,57 @@ span {
   #email-error {
     font-size: .674em;
   }
-}
\ No newline at end of file
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+/* --- MEDIAS IDK IF MAAYOS BA DITO YUNG SA WAVE--- */
+@media (max-width: 1280px) {
+  .index-wave-footer {
+    margin-bottom: -10px;
+  }
+}
+
+@media (max-width: 912px) {
+  .index-wave-footer {
+    margin-bottom: -20px;
+  }
+}
+
+@media (max-width: 540px) {
+  .index-wave-footer {
+    margin-bottom: -30px;
+  }
+}
+
+@media (max-width: 1024px) {
+  .index-wave-footer {
+    margin-bottom: -40px;
+  }
+}
+
+@media (max-width: 912px) {
+  .index-wave-footer {
+    margin-bottom: -50px;
+  }
+}
+/* --- END OF MEDIAS --- */
```

Comment:
this should be consolidated to the media above but for the sake of compliance, this should work.